cloud-director/v27.0.0/release.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  name: cloud-director-27.0.0
+spec:
+  apps:
+  - name: capi-node-labeler
+    version: 0.5.0
+  - name: cert-exporter
+    version: 2.9.1
+    dependsOn:
+    - kyverno-crds
+  - name: cert-manager
+    version: 3.8.1
+    dependsOn:
+    - prometheus-operator-crd
+  - name: chart-operator-extensions
+    version: 1.1.2
+    dependsOn:
+    - prometheus-operator-crd
+  - name: cilium
+    version: 0.25.1
+  - name: cilium-servicemonitors
+    version: 0.1.2
+    dependsOn:
+    - prometheus-operator-crd
+  - name: cloud-provider-vsphere
```

Comment:
We probably also need to make `cluster-cloud-director` consume the release here: https://github.com/giantswarm/cluster-cloud-director/blob/main/helm/cluster-cloud-director/templates/cloud-provider-cloud-director-helmrelease.yaml#L27

Reply:
Oh shoot, yes thanks 🤦🏻

Reply:
The latest test succeeded because `cluster-cloud-director` is always installing `cloud-provider-cloud-director`. So ideally the results stay the same, even with a `cluster-cloud-director` version that consumes releases. :)